kyaml/fn/framework/command/example_test.go

```diff
@@ -0,0 +1,383 @@
+// Copyright 2021 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package command_test
+
+import (
+	"bytes"
+	"fmt"
+
+	"sigs.k8s.io/kustomize/kyaml/fn/framework"
+	"sigs.k8s.io/kustomize/kyaml/fn/framework/command"
+	"sigs.k8s.io/kustomize/kyaml/kio"
+	"sigs.k8s.io/kustomize/kyaml/yaml"
+)
+
+const service = "Service"
+
+// ExampleBuild_modify implements a function that sets an annotation on each resource.
+// The annotation value is configured via ResourceList.FunctionConfig.
+func ExampleBuild_modify() {
+	// create a struct matching the structure of ResourceList.FunctionConfig to hold its data
+	config := &struct {
+		Data map[string]string `yaml:"data" json:"data"`
```

Comment:
Does the `json:"data"` matter? If so, should it also be set in the example below?

Reply:
No, I don't think it matters.